theme/_variables.scss

```diff
@@ -1,8 +1,14 @@
 // Theme Variable overrides
 
+// Body
+$body-bg: #f9f9f9; // Body element's background colour (Bootstrap override).
```

Comment:
@LaurentGoderre In the version of Bootstrap WET currently uses (3.3.1)...
* It's declared here:
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L34
* And used here:
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_scaffolding.scss#L32
* Its value is also re-assigned to some other Bootstrap variables (neither WET nor GCWeb currently re-use them or their parent CSS classes):
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L431
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L436
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L436